src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -3468,31 +3469,58 @@ class C
                 Handle(5, TableIndex.CustomAttribute));
         }
 
-        [Fact]
-        public void ReplaceType()
+        public static string MetadataUpdateOriginalTypeAttributeSource = @"
+namespace System.Runtime.CompilerServices
+{
+    [AttributeUsage(System.AttributeTargets.Class | System.AttributeTargets.Struct, AllowMultiple=false, Inherited=false)]
+    public class MetadataUpdateOriginalTypeAttribute : Attribute
+    {
+        public MetadataUpdateOriginalTypeAttribute(Type originalType) => OriginalType = originalType;
+        public Type OriginalType { get; }
+    }
+}
+";
+
+        public static string BadMetadataUpdateOriginalTypeAttributeSource = @"
+namespace System.Runtime.CompilerServices
+{
+    [AttributeUsage(System.AttributeTargets.Class | System.AttributeTargets.Struct, AllowMultiple=false, Inherited=false)]
+    public class MetadataUpdateOriginalTypeAttribute : Attribute
+    {
+        public MetadataUpdateOriginalTypeAttribute(object originalType) => OriginalType = (Type)originalType;
+        public Type OriginalType { get; }
+    }
+}
+";
+
+        [Theory]
+        [CombinatorialData]
+        public void ReplaceType(bool hasAttribute)
```

Comment:
>ReplaceType [](http://example.com/codeflow?start=20&length=11)

Are we testing when the attribute definition is missing entirely?

Reply:
Yes, other tests in this class that use SemanticEditKind.Replace (e.g. ReplaceType_AsyncLambda)